src/qt/bitcoinstrings.cpp

```diff
@@ -243,15 +243,6 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 "Unsupported chainstate database format found. Please restart with -reindex-"
 "chainstate. This will rebuild the chainstate database."),
 QT_TRANSLATE_NOOP("bitcoin-core", ""
-"Wallet created successfully. The legacy wallet type is being deprecated and "
-"support for creating and opening legacy wallets will be removed in the "
-"future."),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
-"Wallet loaded successfully. The legacy wallet type is being deprecated and "
-"support for creating and opening legacy wallets will be removed in the "
-"future. Legacy wallets can be migrated to a descriptor wallet with "
-"migratewallet."),
-QT_TRANSLATE_NOOP("bitcoin-core", ""
 "Warning: Dumpfile wallet format \"%s\" does not match command line specified "
```

Comment:
You can drop the changes in here, this file will be updated in a translations update. @hebasto might be worth doing one shortly just to cleanup these strings, since they keep being removed in various PRs.